eslintignore not working

I know there has been several issues about it but I can't figure out why my .eslintignore file stop working (with SublimeLinter) since I update my eslint version (to 2.x). It works well if I go back to eslint version 1.10.x.
Is there something I need to change ?

My .eslintignore is near my node_modules folder, at root of my project. I am using a local eslint installation. I have a single folder at the root of my project.

Thanks for your help and for your work.

Is this what you need ?

ST console
```
startup, version: 3103 osx x64 channel: stable
executable: /Applications/Sublime Text.app/Contents/MacOS/Sublime Text
working dir: /
packages path: /Users/arthur/Library/Application Support/Sublime Text 3/Packages
state path: /Users/arthur/Library/Application Support/Sublime Text 3/Local
zip path: /Applications/Sublime Text.app/Contents/MacOS/Packages
zip path: /Users/arthur/Library/Application Support/Sublime Text 3/Installed Packages
ignored_packages: ["Vintage"]
pre session restore time: 0.17006
using gpu buffer for window
startup time: 0.203189
first paint time: 0.295105
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.convert_syntax
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_context_url
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.profile
reloading plugin Default.profile_syntax_definition
reloading plugin Default.quick_panel
reloading plugin Default.run_syntax_tests
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin Python.syntax_test_python
reloading plugin 0_package_control_loader.00-package_control
reloading plugin 0_package_control_loader.01-pygments
reloading plugin 0_package_control_loader.02-bz2
reloading plugin 0_package_control_loader.50-pyyaml
reloading plugin DocBlockr.jsdocs
reloading plugin EditorConfig.EditorConfig
reloading plugin Emmet.emmet-plugin
reloading plugin Markdown Preview.helper
reloading plugin Markdown Preview.markdown_settings
reloading plugin Markdown Preview.markdown_wrapper
reloading plugin Markdown Preview.MarkdownPreview
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
reloading plugin SortBy.sortby
reloading plugin Babel.Babel
reloading plugin SublimeLinter.commands
reloading plugin SublimeLinter.sublimelinter
reloading plugin SublimeLinter-contrib-eslint.linter
SublimeLinter: eslint linter loaded 
plugins loaded
SublimeLinter: debug mode: off 
Emmet: No need to update PyV8
Package Control: Skipping automatic upgrade, last run at 2016-03-17 12:19:24, next run at 2016-03-17 13:19:24 or after

EditorConfig
~/Documents/moon/app/media/js/vendor/modernizr.js
{'charset': 'utf-8',
 'end_of_line': 'lf',
 'indent_size': '4',
 'indent_style': 'tab',
 'insert_final_newline': 'true',
 'tab_width': '4',
 'trim_trailing_whitespace': 'true'}
```

.eslintignore
```
**/*.html
app/index.html.source
app/config.js
app/media/js/bundle.js
app/media/js/vendor/
app/media/views/templates.js
dist/
```

```
MacBook-Pro-Arthur:moon arthur$ npm run build:lint

> moon@1.0.0 build:lint /Users/arthur/Documents/moon
> eslint app/media/js/**/*.js

/Users/arthur/Documents/moon/app/media/js/vendor/modernizr.js
  0:0  warning  File ignored because of a matching ignore pattern. Use --no-ignore to override

✖ 1 problem (0 errors, 1 warning)
```

--- a/linter.py
+++ b/linter.py
@@ -23,7 +23,7 @@ class ESLint(NodeLinter):
     syntax = ('javascript', 'html', 'javascriptnext', 'javascript (babel)',
               'javascript (jsx)', 'jsx-real', 'Vue Component')
     npm_name = 'eslint'
-    cmd = ('eslint', '--format', 'compact', '--stdin', '--stdin-filename', '__RELATIVE_TO_FOLDER__')
+    cmd = ('eslint', '--format', 'compact', '--stdin', '--stdin-filename', '@')
     version_args = '--version'
     version_re = r'v(?P<version>\d+\.\d+\.\d+)'
     version_requirement = '>= 1.0.0'

